setup.cfg

```diff
@@ -5,8 +5,6 @@ universal=0
 ignore =
     # E203: Whitespace before ':'; doesn't work with black
     E203,
-    # E501: line too long
```

Comment:
I copied this ignore set from socorro or wherever. I thought black would occasionally let lines exceed 88 characters depending on their complexity and thus quelling this warning was helpful. I'm not seeing evidence of that in the docs, though, and the line-length section is this one:

https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length

Anyhow, seems like either the situation changed or I misunderstood the docs when I did this.